src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx

```diff
@@ -36,6 +37,36 @@ export default class ClinicalInformationPatientTable extends React.Component<ICl
         }
         return ret;
     }
+    
+    handleClick = (name:string) => {
+        if (this.props.onSelectGenePanel) {
+            this.props.onSelectGenePanel(name);
+        }
+    }
+    
+    renderGenePanelLinks = (genePanels:string) => {
+        const links: (string|JSX.Element)[] = [];
+        const genePanelsArray = genePanels.split(",");
```

Comment:
i think these are genePanelIds, no?   SHould be genePanelIds or genePanelNames.

Reply:
yeah they are indeed gene panel ids, but they are wrapped inside the anchor element and trigger gene panel modal on click so I gave them the name genePanelLinks. If you think it's confusing, I will switch to genePanelIds.